spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminApiKeysController.java

```diff
@@ -256,18 +252,16 @@ public ApiKey getByIdRest(@PathVariable("id") long id) throws IOException {
     public ModelAndView showApiKeyListForLoggedInApiConsumer() {
 
         // Get the ApiKeys for the services consumed by the Logged in User
-        List <ApiKey> apiKeys = apiKeyRepository.findByLoggedInAPIConsumer();
+        List<ApiKey> apiKeys = apiKeyRepository.findByLoggedInAPIConsumer();
 
         // Map the APiKeys to the specific Proxies URLs
-        Map<String, List <String>> proxyURLsPerApiKey = new TreeMap<>();
+        Map<String, List<String>> proxyURLsPerApiKey = new TreeMap<>();
         for (ApiKey apiKey : apiKeys) {
             String apiKeyString = apiKey.getKeyValue();
-            List<String> proxiesUrlList = proxyRepository.customSearchForAllProxiesForGivenServiceApiId(apiKey.getServiceApi().getId())
-                    .stream()
-                    .filter(proxy -> proxy.getPublicUrl() != null)
-                    .filter(proxy -> proxy.getPublicUrl() != "")
-                    .map(Proxy::getPublicUrl)
-                    .collect(Collectors.toList());
+            List<String> proxiesUrlList = proxyRepository
+                    .customSearchForAllProxiesForGivenServiceApiId(apiKey.getServiceApi().getId()).stream()
```

Comment:
This is another example: the newlines were added on purpose to make it easy to read/understand. If possible, the formatter should not join lines here.